test/xpu_api/tuple/tuple.tuple/tuple.creation/tuple_cat1_g.pass.cpp

```diff
@@ -0,0 +1,150 @@
+// -*- C++ -*-
```

Comment:
I guess all of the type checks are done in `tuple_cat.pass.cpp` and this test can be removed.

Reply:
Done: removed